dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -870,6 +871,26 @@
         }
       }
     },
+    "RegexKey": {
+      "description": "The regex key extra data kind.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "RegexKey",
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "The key of the regex pattern used in extracting the entity."
```

Comment:
Could you clarify this? I'm not really sure how this key relates to the regex pattern itself. Looking at some other discriminators doesn't help clarify for me either.

Reply:
The key is the identifier of the regex pattern on the authoring side.
The reason we want to return both the identifier as well as the regex pattern is because the authoring model can be modified even after deploying, so the pattern associated with the same identifier may change on the authoring side.

Reply:
I see. It's basically the name of a named, mutable pattern.

Reply:
In the authoring swagger, this was called `regexKey`. I'd work with whoever the dev is on that and make sure it's consistent in both places.